lib/components/PrimaryButton.jsx

```diff
@@ -73,7 +73,6 @@ PrimaryButton.STATUS_HOVER = 'hover';
 
 PrimaryButton.propTypes = {
   disabled: React.PropTypes.bool,
-  onClick: React.PropTypes.function,
```

Comment:
I had updated to "PropTypes.func" and it still gave me an error. Found out it was a result of another error. SO, fixed all that 👍